doc/sample_java_header.txt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2016 Vivid Solutions.
```

Comment:
Indeed not.  LocationTech is a working group of the Eclipse Foundation.  The Foundation is allows for contributing companies to maintain their copyright as they submit code to projects.